web/e2e/tests/authconnectors.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { expect, test } from '@playwright/test';
+
+import { mockWebAuthn } from '../utils/mockWebAuthn';
+
+test('verify that a user can create and delete an auth connector', async ({
+  page,
+}) => {
+  const { cleanup } = await mockWebAuthn(page);
+
+  await page.goto('');
+
+  await page.getByRole('button', { name: 'Get started' }).click();
+  await page.getByRole('textbox', { name: 'Password', exact: true }).click();
+  await page
+    .getByRole('textbox', { name: 'Password', exact: true })
+    .fill('passwordtest123');
+  await page
+    .getByRole('textbox', { name: 'Password', exact: true })
+    .press('Tab');
+  await page
+    .getByRole('textbox', { name: 'Confirm Password' })
+    .fill('passwordtest123');
+  await page.getByRole('button', { name: 'Next' }).click();
+  await page.getByRole('button', { name: 'Create an MFA Method' }).click();
+  await page.getByRole('button', { name: 'Submit' }).click();
+  await page.getByRole('button', { name: 'Go to Cluster' }).click();
+
+  await page.getByRole('button', { name: "I'll do that later" }).click();
+
+  await page.getByRole('button', { name: 'Zero Trust Access' }).click();
```

Comment:
Uh, I was afraid that we will have to log in before every test.

We should have a utility function that performs all these steps (same on the backend side).